python/cuml/tests/dask/test_dask_logistic_regression.py

```diff
@@ -345,7 +394,11 @@ def imp():
         )
 
     # test predict
-    cu_preds = lr.predict(X_df, delayed=delayed).compute().to_numpy()
+    cu_preds = lr.predict(X_dask, delayed=delayed).compute()
+    if isinstance(cu_preds, cp.ndarray):
+        cu_preds = cp.asnumpy(cu_preds)
+    if not isinstance(cu_preds, np.ndarray):
+        cu_preds = cu_preds.to_numpy()
```

Comment:
What do we expect there? Sparse arrays?

Reply:
cu_preds stores predicted labels in a dense array or cudf.